Change the warning for scan_unevictable_pages to DEBUG, since the warning is just annoying and serves no purpose.

diff --git a/mm/vmscan.c b/mm/vmscan.c
index 313381c..de7701d 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -3493,7 +3493,7 @@ void check_move_unevictable_pages(struct page **pages, int nr_pages)
 
 static void warn_scan_unevictable_pages(void)
 {
-	printk_once(KERN_WARNING
+	printk_once(KERN_DEBUG
 		    "The scan_unevictable_pages sysctl/node-interface has been "
 		    "disabled for lack of a legitimate use case.  If you have "
 		    "one, please send an email to linux-mm@kvack.org.\n");
diff --git a/net/ipv6/ndisc.c b/net/ipv6/ndisc.c
index 319cd57..5c6399c 100644
--- a/net/ipv6/ndisc.c
+++ b/net/ipv6/ndisc.c
@@ -1761,7 +1761,7 @@ static void ndisc_warn_deprecated_sysctl(struct ctl_table *ctl,
 	static int warned;
 	if (strcmp(warncomm, current->comm) && warned < 5) {
 		strcpy(warncomm, current->comm);
-		printk(KERN_WARNING
+		printk(KERN_DEBUG
 			"process `%s' is using deprecated sysctl (%s) "
 			"net.ipv6.neigh.%s.%s; "
 			"Use net.ipv6.neigh.%s.%s_ms "
